.github/workflows/continuous-integration.yml

```diff
@@ -50,3 +50,22 @@ jobs:
         popd
     - name: Build with Maven
       run: mvn -Dlogging.config.file=\${maven.multiModuleProjectDirectory}/logging.ci.properties verify -B
+  deploy:
+    needs: build
```

Comment:
I think that this gets complicated because you need to use the artifact created by the `build`.